### ISSUE TITLE
Names of team members to link to their user profile page

### ISSUE THREAD

**Issue by sarahboyce** [django/djangoproject.com#1738]
If you go to the teams page you see a lot of names of different folks: https://www.djangoproject.com/foundation/teams/

I would like these names to be a link to their user profile
So for example, my name would link to https://www.djangoproject.com/~sarahboyce/

**Comment by bmispelon** [comment]
I'm assuming this is the forum thread you meant? https://forum.djangoproject.com/t/what-is-the-technical-team-and-the-technical-advisory-team/36455

**Comment by sarahboyce** [comment]
> I'm assuming this is the forum thread you meant? https://forum.djangoproject.com/t/what-is-the-technical-team-and-the-technical-advisory-team/36455

Yes, thank you! I have updated the proposal quite a bit also :+1:

**Comment by jefftriplett** [comment]
This is shiny and I like it. Great idea and proposal! 

[This can be a different ticket/issue/patch]

Is there a good or reasonable alternative to Gravatar which I see linked up in the sidebar? Given the recent Automattic meltdown, I deleted my account and one use or advocate anything they touch. Which is a shame. I don't think this image quite captures how pale I really am. https://www.djangoproject.com/~jefftriplett/

**Comment by pauloxnet** [comment]
> Is there a good or reasonable alternative to Gravatar which I see linked up in the sidebar? Given the recent Automattic meltdown, I deleted my account and one use or advocate anything they touch. Which is a shame. I don't think this image quite captures how pale I really am. https://www.djangoproject.com/~jefftriplett/

First ideas that came in mind: the avatar from the GitHub user, since all contributors I think already have one account over there. Or we can add an `URLField` in the profile so you can choose. Pros and cons in both solutions.

**Comment by marksweb** [comment]
You can fetch profile data for a user from this endpoint;

https://api.github.com/users/marksweb
